### PR TITLE
Update resource gallery

### LINKED ISSUE #439: Resource Gallery submission automation is broken
The automation that generates PRs to add new entries to the Resource Gallery is not working.

#397 filled out the form and triggered #398. #398 removes all content from `portal/resource_gallery.yaml` and replaces it with the new submission, but the new submission is not in YAML format.

I just tried repeating the same entry in #437 and #438 and got the same result.

I don't think this is related to the new theme. #397 was submitted on 2/9/2024, before the theme updates. #437 was from today.

### PR DESCRIPTION
**Pull request by jukent**
Closes #439

### CODE REVIEW

File: .github/workflows/update-resource-gallery.yaml [hunk 1]
```diff
@@ -93,12 +93,17 @@ jobs:
 
           submission_file = 'resource-gallery-submission-input.json'
           resource_gallery_file = 'portal/resource_gallery.yaml'
-
+
```
Comment (jukent) [inline .github/workflows/update-resource-gallery.yaml]:
```suggestion

```

**Review by erogluorhan — APPROVED** [review]
This was much needed; thanks a lot for doing it!

**Comment by jukent** [comment]
Seems to still be having issues: https://github.com/ProjectPythia/projectpythia.github.io/actions/runs/12240686083/job/34144102576

**Comment by erogluorhan** [comment]
> Seems to still be having issues: https://github.com/ProjectPythia/projectpythia.github.io/actions/runs/12240686083/job/34144102576

It seems to be due to a validation issue this time though (previously validation was passed and PR was created but the PR's changed file was ill-formatted and mistaken content). The validation failure is probably due to a format problem in the [issue you created](https://github.com/ProjectPythia/projectpythia.github.io/issues/470). Please compare with this [older issue](https://github.com/ProjectPythia/projectpythia.github.io/issues/455) and try to match its format/content

**Comment by jukent** [comment]
@erogluorhan I actually copied that issue to create the test. But I'm realizing now that I copied the formatted content instead of the raw.

**Comment by jukent** [comment]
Also that old issue is triggering validation errors now (I think because Michaela's affiliation isn't a string). I'm not sure why it is more picky now than it used to be.

It would be nice if there was some sort of comment when validation checks fail.

**Pull request merged**